examples/typescript/src/index.ts

```diff
@@ -1,18 +1,18 @@
-import * as spsFrontend from "@tensorworks/libspsfrontend";
+import * as libspsfrontend from "@tensorworks/libspsfrontend";
```

Comment:
I personally prefer the other way where you do like:

`import Config from "@tensorworks/libspsfrontend"`

and can remove the `libspsfrontend` prefix when you make the calls, e.g.

`const config = new libspsfrontend.Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });`

becomes

`const config = new Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });`

Reply:
This has been resolved in 0cbcf882f39e1ca387c18253c8e24afa4fd0e7a4